.gitignore

```diff
@@ -1,4 +1,4 @@
 *.log
 .DS_Store
-node_modules
+node_modules/
```

Comment:
Why did you add this? If it's causing Preflight to fail without it, we should fix this in Preflight...

Reply:
I've removed the slash again in `main`. For me, Preflight runs through.

Reply:
If it doesn't for you (on Windows), then let's fix the issue in Preflight, not change the `.gitignore`.

Reply:
i think was failin for other reason an di thought was this, i changed to test and i forgot to change it back